### PR TITLE
Pressure_resistance Tweaks

### PR DESCRIPTION
**Pull request by Cheridan**
-Adjusts pressure resistance, which determines how easily things can get pushed around by atmos.
-Most items (handheld stuff) are less resistant, while most larger items (mobs, structures, machinery) are more resistant.
-Ore boxes specifically are very resistant because fuck that.

-Removes the cooldown on air pushing stuff, which should make it feel more 'responsive', as opposed to the current push->pause->push->pause effect it does now.
-MAGBOOTS/NOSLIPS NOW PREVENT SPACEWIND PUSHING! !!!
-Adds a little wooshy breeze sound to when you would get pushed

### CODE REVIEW

**Comment by duncathan** [comment]
do stabilized jetpacks prevent spacewind too?

**Comment by PKPenguin321** [comment]
>MAGBOOTS/NOSLIPS NOW PREVENT SPACEWIND PUSHING! !!!

nice
>Removes the cooldown on air pushing stuff, which should make it feel more 'responsive', as opposed to the current push->pause->push->pause effect it does now.

terrifying
>Adds a little wooshy breeze sound to when you would get pushed

more responsiveness is a good thing, :+1:

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -29,6 +29,12 @@
 		return 0
 	.=..()
 
+/mob/living/carbon/human/experience_pressure_difference()
+	playsound(src, 'sound/effects/space_wind.ogg', 50, 1)
+	if(shoes.flags&NOSLIP)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human_movement.dm]: this means the janitor will be immune to spacewind---which makes no sense; shouldn't this check for MAGPULSE on magboots, specifically?

Comment (optimumtact) [inline code/modules/mob/living/carbon/human/human_movement.dm]: new flag

Comment (Cheridan) [inline code/modules/mob/living/carbon/human/human_movement.dm]: blehhhhh hh

Galoshes got really nice rubberized grips!

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Currently there's really no reason, at all, to get magoots as a syndi agent---with this, there will never be a reason, as no-slip shoes would be cheaper (and not slow you down...why do syndi-mags suck, again?)

Yes yes, hypothetically Lord Singulo and junk, but that never really happens.

**Comment by KorPhaeron** [comment]
"Removes the cooldown on air pushing stuff, which should make it feel more 'responsive', as opposed to the current push->pause->push->pause effect it does now"

Can you not get inside from a hull breach now?

**Comment by optimumtact** [comment]
while most larger items (mobs, structures, machinery) are more resistant.

**Comment by Cheridan** [comment]
@KorPhaeron You still can. Using an airlock as an example, it's a little harder at first as the air pushes you out but then the air level equalizes and the increased resistance means you can get in, without the tiny little gusts blowing you back out over and over again.

**Comment by RemieRichards** [comment]
I tried to do the magboot thing but realised we'd need a new flag (NOSLIP is not a good replacement as has already been stated) and our flag system is already 2 flags over the maximum you can have in a bitfield (with 2 duplicates (4 flags sharing 2 numbers))

**Comment by MrStonedOne** [comment]
So, I added that cooldown for a reason. While yes, you can still get back inside normal airlocks, you can prevent anybody from getting on the shuttle by rcding the floor in front of the shuttle airlocks.

without the airlock -> floor -> airlock pattern, the air takes too long to drain out, and it flat out becomes impossible, even with speedups, to get in from a hull breach or even anywhere space is next to not space where there is an actual substantial amount of tiles with air still draining.


Test it on the shuttle, rcd the floor under the airlock in escape, call the shuttle, check antag edit its time, and try to get on it. it's fucking impossible, even with speedup chems.

**Comment by Cheridan** [comment]
Just tested. It's completely possible to get onto the shuttle in that situation, even after standing on the space tile until I was fully cold. Again, humans are more resistant to smaller atmos pushes now.

**Comment by Razharas** [comment]
NOSLIP immunity is silly everything else is ready to merge

**Pull request merged**